probe/docker/registry.go

```diff
@@ -472,13 +472,26 @@ func (r *registry) WalkNetworks(f func(docker_client.Network)) {
 	}
 }
 
-// ImageNameWithoutVersion splits the image name apart, returning the name
-// without the version, if possible
-func ImageNameWithoutVersion(name string) string {
+// ImageNameParts returns parts of the full image name (image name, image tag).
+func ImageNameParts(name string) []string {
 	parts := strings.SplitN(name, "/", 3)
 	if len(parts) == 3 {
 		name = fmt.Sprintf("%s/%s", parts[1], parts[2])
 	}
-	parts = strings.SplitN(name, ":", 2)
-	return parts[0]
+	return strings.SplitN(name, ":", 2)
+}
+
+// ImageNameWithoutVersion splits the image name apart, returning the name
+// without the version, if possible
+func ImageNameWithoutVersion(name string) string {
```

Comment:
Also I would keep the naming to be always "Tag" rather then Version, to avoid confusion. So `ImageName` and `ImageNameNoTag` for example, WDYT?